did/model_test.go

```diff
@@ -3,9 +3,10 @@
 package did
 
 import (
-	"encoding/json"
 	"testing"
 
+	"github.com/goccy/go-json"
```

Comment:
solid choice! being able to pass Context.context into Marshal and Unmarshal is awesome